src/Workspaces/Core/Portable/Storage/SQLite/v2/Interop/SqlConnection.cs

```diff
@@ -299,7 +299,7 @@ public Optional<Stream> ReadDataBlob_MustRunInTransaction(Database database, Tab
         }
 
         [PerformanceSensitive("https://github.com/dotnet/roslyn/issues/36114", AllowCaptures = false)]
-        public Optional<Checksum.HashData> ReadChecksum_MustRunInTransaction(Database database, Table table, long rowId)
+        public Optional<Checksum> ReadChecksum_MustRunInTransaction(Database database, Table table, long rowId)
```

Comment:
>Optional<Checksum> [](http://example.com/codeflow?start=15&length=18)

not asking to change, but this is essentially just "Checksum?", except in roslyn specific terms, right? Which could instead just be Checksum.Null? But it looks like this might propagate through the code as Optional, so probably not worth changing now.

Reply:
yeah, the Optional is relevant here as it's part of pattern around how the storage infra states that it couldn't even find the info at all.  Don't want to touch that right now :)